README.rst

```diff
@@ -350,6 +350,62 @@ shows the different configuration options available:
         # image is passed to subsequent steps.
         import: path/to/image/archive.tar
 
+        # Specify the secrets that should be used when building your image,
+        # similar to the --secret option used by Docker
+        # More info about secrets: https://docs.docker.com/build/building/secrets/
+        secrets:
+          # Example of a secret that is a file
+          - id=secret1,src=<path to the secret file>
+          # Example of a secret that is an environment variable
+          - id=secret2,env=<environment variable name>
+
+
+.. _Build Secrets:
+
+Build Secrets
+=============
+
+Buildrunner supports specifying secrets that should be used when building your image,
+similar to the --secret option used by Docker. This is done by adding the ``secrets``
+section to the ``build`` section. This is a list of secrets that should be used when
+building the image. The string should be in the format of ``id=secret1,src=<location of the file>``
+when the secret is a file or ``id=secret2,env=<environment variable name>`` when the secret is an environment variable.
+This syntax is the same as the syntax used by Docker to build with secrets.
+More info about building with secrets in docker and the syntax of the secret string
+see https://docs.docker.com/build/building/secrets/.
+
+In order to use secrets in buildrunner, you need to do the following:
+
+#. Update the buildrunner configuration file
+    * Set ``use-legacy-builder`` to ``false``
```

Comment:
not applicable for builds using `platforms`, right?

Reply:
That is correct. I didn't include that distinction here. I tried to keep it simple. But maybe I should.

Reply:
I am making a code update so that platforms doesn't need to set `use-legacy-builder`

Reply:
I updated this to not that if it has platforms it doesn't need `use-legacy-builder`